lightning_client.go

```diff
@@ -683,6 +683,24 @@ const (
 	InitiatorBoth
 )
 
+// ForceCloseAnchorState indicates the resolution state for the anchor.
+// There are three resolution states for the anchor: limbo, lost and recovered.
+type ForceCloseAnchorState uint8
+
+const (
+	// ForceCloseAnchorStateLimbo is set if the recovered_balance is zero
+	// and limbo_balance is non-zero.
+	ForceCloseAnchorStateLimbo ForceCloseAnchorState = iota
```

Comment:
Hmm, this seems brittle if we use `iota` for types that we declare in the RPC. Sure, we most likely never change existing RPC values. But maybe we could still just do a shorter const alias here? e.g. `ForceCloseAnchorStateLimbo ForceCloseAnchorState = lnrpc.PendingChannelsResponse_ForceClosedChannel_LIMBO`?

Reply:
Yeah agree. Changed to your suggestion.